client/src/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
-import GlobalStyle from './globalStyles';
+import GlobalStyle from './GlobalStyles';
```

Comment:
I would name it `GlobalStyles` when the name of your file is also `GlobalStyles`. ;)